### DISCUSSION TITLE
data.py

### DISCUSSION THREAD

**Discussion by incoghardo** [EodHistoricalData/python-eodhistoricaldata#25] [Q&A]

**Comment by deios0** [comment]
I do not see any question here.

**Reply by incoghardo** [reply]
Yes sorry I tried to delete it after finding my answer. I was asking about adding code for futures data but have switched to a different data provider

**Comment by deios0** [comment]
Thank you for the feedback, I believe we do provide the same data for futures.
Anyway, thank you and looking forward to seeing you again!

**Reply by incoghardo** [reply]
Sorry let me clarify what I am saying, the data.py script did not appear to have code for pulling futures data.

**Comment by deios0** [comment]
The futures data is actually the same as EOD data, you just should use GC.COMM, ticker code for example in EOD API, to get the data for these futures.

**Reply by incoghardo** [reply]
Ah I see now. I missed it at the bottom where the indices and exchanges are defined but it will work. Also you can change a few words in the get eod data function to get intraday data if you wanted to add that to your wrapper.

**Comment by deios0** [comment]
Yes, we will check it
